resources/stats/stats.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+cd "$THIS_SCRIPT_PATH"
+
+platforms=(android/ ios/ linux/ mac/ web/ windows/ developer/ core/ common/models/)
+
+startdate=2022-05-01
+stable=stable-15.0
```

Comment:
Do we want stable-16.0?

Reply:
Yeah but not important, this will need checking before I use it again anyway.